gherkin-utils/java/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.cucumber</groupId>
+  <artifactId>gherkin-utils</artifactId>
+  <version>21.0.1-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <name>Gherkin Utils</name>
+  <description>Gherkin utils for doing things such as walking a Gherkin tree or pretty printing Gherkin files.
+  </description>
+  <url>https://github.com/cucumber/common</url>
+
+  <properties>
+    <project.Automatic-Module-Name>io.cucumber.gherkin-utils</project.Automatic-Module-Name>
+    <maven.compiler.source>8</maven.compiler.source>
+    <maven.compiler.target>8</maven.compiler.target>
+  </properties>
+
+  <scm>
+    <connection>scm:git:git://github.com/cucumber/common.git</connection>
+    <developerConnection>scm:git:git@github.com:cucumber/common.git</developerConnection>
+    <url>git://github.com/cucumber/common.git</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.cucumber</groupId>
```

Comment:
This should be `test` scope.

Please add a runtime dependency on `messages`

Reply:
i could not make this change, it caused project not to compile due to missing dependencies during compile time.